### ISSUE TITLE
The app doesn't keep the screen on while reading

### ISSUE THREAD

**Issue by Palloxin** [LNReader/lnreader#645]
### Steps to reproduce

1. open a chapter
2. click "next chapter", < or >
3. don't touch for 23-25 (might be more) seconds the screen while reading a chapter with battery-save mode on (it enables screen-turn-off timer)

### Expected behavior

Screen is kept on while reading

### Actual behavior

the video speed is accelerated (x3)

https://github.com/LNReader/lnreader/assets/75091899/7f74c894-b2c1-45dd-a880-5c9f41062f5e

EDIT: I just noticed the screen recorder doesn't catch the variation of luminosity so the video is useless, sorry

EDIT2: I changed the video with one made with another phone on a phone so the change of luminosity can be detected.

### LNReader version

1.1.18

### Android version

Android 8 miui 8

### Device

huawei p8 lite 2018

### Other details

EDIT2: i did multiple videos and i noticed that the app dimmed only after i went to the next chapter.
My first attempt i just waited on the first chapter of "resume" button without changing chapter and the app didn't dim at all after a minute.
Then i changed chapter with the button > and some time after the app dimmed (video)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.15](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Palloxin** [comment]
I forgot to say i also added lnreader to the list of app that can Ignore battery optimization but it still happens

**Comment by Palloxin** [comment]
Here is undeniable proof that the moment i click "next chapter" or ">" and go to the next chapter the "keep screen on" doesn't work anymore.
The videos are split because github doesnt accept a long one somehow.
https://github.com/LNReader/lnreader/assets/75091899/1c8f05e5-9279-4956-b7f3-8b5f5f985389

https://github.com/LNReader/lnreader/assets/75091899/b1d0e5cb-908a-4256-9fb1-723a92af86e9

Tge last 30 sec the speed of the video is x1 while the rest is x3

**Comment by Palloxin** [comment]
The bug dynamic seems related ti #719 . Once the reader leaves the first instance clickimg next chapter,< or > the feature will not apply.
